home/templates/home/home_page.html

```diff
@@ -4,7 +4,8 @@
 {% block content %}
```

Comment:
Instead of using the `content` template block, you need to use the `before_content` block.

Reply:
I will do it